site/releases/2024-feb-14/highlights.qmd

```diff
@@ -0,0 +1,236 @@
+
+## Enhancements
+
+<!---Panchicore/sc 2799/model owner should be able to edit model by @panchicore in [#562](https://github.com/validmind/frontend/pull/562)--->
+Model owner can edit model details page
+: Model owners can now edit the values for fields displayed on the model details page. Previously it was only possible to edit custom fields defined by your organization.
+
+<!---Reducing load time by using localstorage for Auth0 by @even-steven in [#570](https://github.com/validmind/frontend/pull/570)--->
+Reduced load times for the ValidMind Platform UI
+: We reduced the loading times of our Platform UI to improve the user experience.
+
+<!---Added support for filtering model inventory by developers / validators by @even-steven in [#563](https://github.com/validmind/frontend/pull/563)--->
+Added support for filtering model inventory by developers / validators
+: Enhanced the Model Inventory filtering to allow filtering by Developers and Validators.
+
+<!---Support for custom model fields by @panchicore in [#509](https://github.com/validmind/frontend/pull/509)--->
+Support for custom model fields in the model inventory
+: The model inventory has been updated to allow organizations to add custom fields. This enhancement enables administrators to customize the model inventory according to your specific organizational needs.
+
+The initial release supports the following field types:
+
+- Single Line Text
+- Long Text
+- Single Select
+- Multiple Select
+- Checkbox
+- Number
+- URL
+- Date
+- Date Time
+- Email
+- Linked Record to User
+
+<!---Adds toggle to see only mentions in Recent Activity > Comments by @gtagle in [#525](https://github.com/validmind/frontend/pull/525)--->
+Filter for mentions in recent activity comments
+: We implemented a toggle feature in the **Recent Activity** > **Comments** section to filter and display only specific mentions. By default, all comments where the logged-in user has been tagged are displayed by this filter. 
+
+<!---Add CKEditor support to finding description and remediation plan - Introduce the `RichTextContentEditor` component by @panchicore in [#542](https://github.com/validmind/frontend/pull/542)--->
+Expanded rich-text editor support
+: Forms in the **Findings** and **Validation Report** sections now support the rich-text editor interface. This support enables you to use the editor for your finding descriptions and remediation plans, for example. 
+
+## Bug fixes
+
+<!---Invalid content blocks create errors in run documentation by @cachafla in [#326](https://github.com/validmind/developer-framework/pull/326)--->
+Invalid content blocks create errors for `run_documentation_tests()`
+: Fixed an issue where using an invalid test identifier would prevent `run_documentation_tests()` from running all available tests. The full test suite now runs as expected, even when an invalid test identifier is specified.
```

Comment:
@cachafla  - does the framework throw some message to the user when an invalid test identifier is specified? If yes, I would clarify here

Reply:
Yes. It throws an error saying it could not find the test specified in the template.